ios/RNSplashScreen.m

```diff
@@ -36,7 +36,16 @@ + (void)show {
 + (void)hide {
   // let's try to hide, even if showing == false, ...just in case
 
-  UIImageView *imageView = (UIImageView *)[UIApplication.sharedApplication.keyWindow.subviews.lastObject viewWithTag:RNSplashScreenOverlayTag];
+  NSArray *subviews = UIApplication.sharedApplication.keyWindow.subviews;
+  UIImageView *imageView = (UIImageView *)[subviews.lastObject viewWithTag:RNSplashScreenOverlayTag];
+
+  #ifdef DEBUG
+  if (imageView == nil) {
```

Comment:
```suggestion
  if (imageView == nil && subviews.count > 1) {
```

Reply:
Forgot we try to hide even if splash is already hidden 🤦